src/docs.js

```diff
@@ -1,4 +1,35 @@
-function documenter() {
+let tar = require('tar-stream');
+
+async function documenter(options = {}) {
+  // print schemas
+  console.log(options.schemas);
```

Comment:
Set default options if any:
```js
options = _.defaults({}, options, {
  schemas: [],
});
```

Make some asserts on your options...
```js
assert(options.schemas, 'options.schemas must be given');
assert(options.schemas instanceof Array, 'options.schemas must be an array');
```
Simple stupid asserts like that will save us a lot of trouble in the long run...